service/src/main/java/bio/terra/cbas/controllers/RunSetsApiController.java

```diff
@@ -484,7 +484,8 @@ private List<RunStateResponse> buildInputsAndSubmitRun(
             cromwellService.submitWorkflow(rawMethodUrl, workflowInputs, workflowOptionsJson);
 
         runStateResponseList.add(
-            storeRun(runId, workflowResponse.getRunId(), runSet, record.getId(), UNKNOWN, null));
+            storeRun(
+                runId, workflowResponse.getRunId(), runSet, record.getId(), INITIALIZING, null));
```

Comment:
When we first store the run in our database, I think it should have the INITIALIZING state. If there is an issue submitting the workflow to cromwell above, this code will not be reached as we won't even have a workflowResponse. In this case, the status will be set to SYSTEM_ERROR. By setting it to initializing, we are more accurately reflecting our current knowledge of the state - we have submitted the workflow to cromwell, no issues yet, but it hasn't started running to our knowledge.